utils/build-presets.ini

```diff
@@ -681,7 +681,7 @@ enable-new-runtime-build
 
 # Escalate certain C++ warnings to errors for Swift.
 extra-swift-cmake-options=
-   -DCMAKE_CXX_FLAGS="-Werror=unused"
```

Comment:
I wonder if we should move this to the `buildbot_incremental_base` preset? The smoketest preset only gets run if someone explicitly asks for a smoke test, but not if they only ask for PR testing.

Reply:
Alternatively, it might be good to spell out the flags because at this point, I have no idea what the existing flags are, or where they're coming from. These preset files don't make it easy to track down where flags are coming from.